src/main/native/cpp/rubik_jni.cpp

```diff
@@ -469,22 +469,26 @@ Java_org_photonvision_rubik_RubikJNI_detect
 
   std::memcpy(TfLiteTensorData(input), rgb.data, TfLiteTensorByteSize(input));
 
-  // Start timer for benchmark
+// Start timer for benchmark
+#ifndef NDEBUG
```

Comment:
Love (don't) a double negative. Do we get a DEBUG macro? Could we if DEBUG instead?